CMakeLists.txt

```diff
@@ -163,8 +163,10 @@ set(SKILLS_SOURCES
         ${PROJECT_SOURCE_DIR}/src/skills/Wait.cpp
         ${PROJECT_SOURCE_DIR}/src/skills/KickTo.cpp
         ${PROJECT_SOURCE_DIR}/src/skills/formations/StopFormation.cpp
-        ${PROJECT_SOURCE_DIR}/src/skills/formations/BallPlacementFormation.cpp
-        )
+        ${PROJECT_SOURCE_DIR}/src/skills/formations/BallPlacementFormation.cpp)
+
+set(NEW_SKILLS_SOURCES
```

Comment:
NEW_SKILL_SOURCES -> SKILL_SOURCES (unless it already exists

Reply:
SKILL_SOURCES already exist, for clarity the NEW_SKILL_SOURCES will be used until the old skills are deprecated